web/modules/custom/ai_screening_fixtures_base/src/Helper/Helper.php

```diff
@@ -80,8 +60,28 @@ public function createImagesFromAssets(): array {
    *   The contents of the file.
    */
   public function getText(string $filename): ?string {
-    $texts_source_path = $this->pathResolver->getPath('module', 'ai_screening_fixtures_base') . '/assets/texts';
-    return file_get_contents($texts_source_path . '/' . $filename) ?? NULL;
+    $textsSourceFilePath = __DIR__ . '/../assets/texts/' . $filename;
+    if (!file_exists($textsSourceFilePath)) {
+      throw new FileNotFoundException($textsSourceFilePath);
+    }
+
+    return file_get_contents($textsSourceFilePath) ?? NULL;
+  }
+
+  /**
+   * Log in a user.
+   *
+   * @param int $uid
+   *   THe user id.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function userLogin(int $uid): void {
+    $user = $this->entityTypeManager->getStorage('user')->load($uid);
+    if ($user instanceof UserInterface) {
+      user_login_finalize($user);
+    }
```

Comment:
Do you plan on using this function where we don't have a `UserInterface` instance when calling it? If not, consider passing the `UserInterface`:

```suggestion
  public function userLogin(UserInterface $user): void {
    user_login_finalize($user);
  }
```